cachito/web/app.py

```diff
@@ -110,6 +115,14 @@ def create_app(config_obj=None):
 
     app.register_blueprint(docs)
     app.register_blueprint(api_v1, url_prefix="/api/v1")
+
+    path = Path(__file__).parents[1].absolute()
```

Comment:
We could change this to `Path(__file__).parent.absolute()` and then reference the file with `f"{path}/static/api_v1.yaml"`